script/Arcueid/Day 12/12_00_ARC12_2.txt

```diff
@@ -421,6 +425,7 @@ I don't think I've ever heard something so viscerally repulsive before.
 -- 紙は年月に霞むほど<脆|もろ>い。
 // It's a little weird to say that paper grows _blurry_ with age.
 // Normally you'd say that of the ink. Kept noun changed verb, but could go either way.
+// group pr check: ask petrikow again
```

Comment:
IIRC context was that this is comparative - rather than "the more the paper ages the more it fades", it's more like the
> Pages as muddled and vague as the passage of time itself.